documentation/src/docs/beginner_dev_guide.md

```diff
@@ -0,0 +1,205 @@
+<!--
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
```

Comment:
The copyright header has changed, please fix it.

Reply:
Thanks, fixed it.